### PR TITLE
Update nightly feed

### LINKED ISSUE #6347: [Bug] Nightly builds are not updated
### Description

Nightly builds are not updated each day as the name suggests.

### Steps to Reproduce

Check [Latest nightly build](https://myget.org/feed/xamarinforms-ci/package/nuget/Xamarin.Forms)

### Expected Behavior

The automatic build system should release a nightly build each day.

### Actual Behavior

The latest nightly build has been submitted Sat, 27 Apr 2019.

### Basic Information

Building XF has some complexity, especially for Mac users and since the usage of Azure Devops for this task has not been explained yet. Thus verifying the latest improvements/bug fixes can be painful.

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change ###

Update with nightly feed from Azure 

Known issues:
Current this is a azure static feed since there's no server side code we don't have search capabilities 
### Issues Resolved ### 

- fixes #6347

### API Changes ###

 
 None

### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- Android
- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### PR Checklist ###

- [ ] Has automated tests 
- [ ] Rebased on top of the target branch at time of PR
- [ ] Changes adhere to coding standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ Read more about the platform at https://www.xamarin.com/forms.
 
 ## Packages ##
 
-Platform/Feature               | Package name                              | Stable (3.0.0 branch)     |Nightly Feed [MyGet](https://www.myget.org/F/xamarinforms-ci/api/v2)  (master branch)
+Platform/Feature               | Package name                              | Stable (3.0.0 branch)     |Nightly Feed [Azure](https://nugetized.blob.core.windows.net/xamforms-nightly/index.json)  (master branch)
```
Comment (rmarinho) [inline README.md]:
```suggestion
Platform/Feature               | Package name                              | Stable (3.0.0 branch)     |Nightly Feed [Azure](https://aka.ms/xf-ci/index.json)  (master branch)
```

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ If you want to use the latest dev build then you should read [this blog post](ht
   <configuration>
     <packageSources>
       <clear />
-      <add key="xamarin-ci" value="https://www.myget.org/F/xamarinforms-ci/api/v2" />
+      <add key="xamarin-ci" value="https://nugetized.blob.core.windows.net/xamforms-nightly/index.json" />
```
Comment (rmarinho) [inline README.md]:
```suggestion
      <add key="xamarin-ci" value="https://aka.ms/xf-ci/index.json" />
```

File: README.md [hunk 3]
```diff
@@ -13,12 +13,12 @@ Read more about the platform at https://www.xamarin.com/forms.
 
 ## Packages ##
 
-Platform/Feature               | Package name                              | Stable (3.0.0 branch)     |Nightly Feed [MyGet](https://www.myget.org/F/xamarinforms-ci/api/v2)  (master branch)
+Platform/Feature               | Package name                              | Stable (3.0.0 branch)     |Nightly Feed [Azure](https://aka.ms/xf-ci/index.json)  (master branch)
```
Comment (StephaneDelcroix) [inline README.md]: Stable is no longer 3.0.0

Comment (StephaneDelcroix) [inline README.md]: the nightly column is empty

Comment (rmarinho) [reply README.md]: yeah we don t have badges yet

**Pull request merged**